.github/workflows/wildfly-it-tests-bootable.yml

```diff
@@ -0,0 +1,41 @@
+name: WildFly Bootable JAR Integration Tests
+
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+    branches: [ 'master' ]
+
+jobs:
+  wildfly-it-tests:
+    if: '! github.event.pull_request.draft'
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest']
+        java-version: ['17', '21']
+        java-distribution: ['adopt']
+    steps:
+      - name: Checkout eap-microprofile-test-suite
+        uses: actions/checkout@v4
+      - name: Set up JDK ${{ matrix.java-distribution }} ${{ matrix.java-version }}
+        uses: actions/setup-java@v4
+        with:
+          java-version: ${{ matrix.java-version }}
+          distribution: ${{ matrix.java-distribution }}
+          cache: 'maven'
+      - name: Build and run integration tests (${{ matrix.java-distribution }} ${{ matrix.java-version }}) against latest WildFly, on ${{ matrix.os }}
+        if: ${{ matrix.os  == 'ubuntu-latest' }}
+        run: mvn clean verify -fae "-Dts.bootable" "-Dcurrent-execution.excluded-groups=org.jboss.eap.qe.microprofile.health.junit.HealthWithFaultToleranceTests,org.jboss.eap.qe.microprofile.health.junit.ManualTests"
```

Comment:
would you consider to add `--batch-mode` param to all mvn calls to avoid unnecessary download progress lines in build log?

Reply:
+1, it was there initially, but I must have lost it along the way. Fixed, thanks.